code/modules/admin/topic.dm

```diff
@@ -426,6 +426,8 @@ Status: [status ? status : "Unknown"] | Damage: [health ? health : "None"]
 				newmob = M.change_mob_type(/mob/living/carbon/xenomorph/boiler, location, null, delmob)
 			if("crusher")
 				newmob = M.change_mob_type(/mob/living/carbon/xenomorph/crusher, location, null, delmob)
+			if("gorger")
+				newmob = M.change_mob_type(/mob/living/carbon/xenomorph/gorger, location, null, delmob)
```

Comment:
doesnt this also need an admin ui update

Reply:
removed for now